string road_id
int lane_id
monodrive_msgs/Waypoint[] waypoints